test/e2e/specs/realrp/realrp.go

```diff
@@ -2,59 +2,55 @@ package realrp
 
 import (
 	"context"
-	"errors"
 	"fmt"
+	"net/http"
 	"os"
-	"path"
+	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
 	"github.com/Azure/go-autorest/autorest/to"
 
-	fakerp "github.com/openshift/openshift-azure/pkg/fakerp/shared"
-	"github.com/openshift/openshift-azure/pkg/util/managedcluster"
+	"github.com/openshift/openshift-azure/pkg/fakerp/client"
 	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/util/log"
 )
 
-const (
-	fakepubkey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC7laRyN4B3YZmVrDEZLZoIuUA72pQ0DpGuZBZWykCofIfCPrFZAJgFvonKGgKJl6FGKIunkZL9Us/mV4ZPkZhBlE7uX83AAf5i9Q8FmKpotzmaxN10/1mcnEE7pFvLoSkwqrQSkrrgSm8zaJ3g91giXSbtqvSIj/vk2f05stYmLfhAwNo3Oh27ugCakCoVeuCrZkvHMaJgcYrIGCuFo6q0Pfk9rsZyriIqEa9AtiUOtViInVYdby7y71wcbl0AbbCZsTSqnSoVxm2tRkOsXV6+8X4SnwcmZbao3H+zfO1GBhQOLxJ4NQbzAa8IJh810rYARNLptgmsd4cYXVOSosTX azureuser"
-)
-
-var _ = BeforeSuite(func() {
-	if os.Getenv("AZURE_REGION") == "" {
-		// Set AZURE_REGION from the manifest if it is not set and the manifest exists.
-		dataDir, err := fakerp.FindDirectory(fakerp.DataDirectory)
-		if err == nil {
-			oc, err := managedcluster.ReadConfig(path.Join(dataDir, "manifest.yaml"))
-			if err == nil {
-				os.Setenv("AZURE_REGION", oc.Location)
-			}
-		}
-	}
-})
-
-var _ = Describe("Resource provider e2e tests [Real]", func() {
+var _ = Describe("Resource provider e2e tests [Default][Real]", func() {
 	var (
 		ctx = context.Background()
 		cli *azure.Client
+		cfg *client.Config
 	)
 
-	BeforeEach(func() {
+	// NOTE: Ensure this is always the first test in the [Default][Real] spec!
+	It("should deploy a cluster using the production RP", func() {
```

Comment:
The alternative solution to this that I though is to create a new `test/e2e/specs/realrp/default.go` that would simply deploy a prod cluster such as the vnet peering test is doing and then somehow reuse all tests in this file in both `default.go` and `vnetpeering.go`. That was problematic because the vnet peering test is cleaning up itself in a `AfterEach` function. Maybe we want to fix this in a follow up, didn't seem pressing to do here.

Reply:
I'm fine removing the clean up of itself.  I only did that because I wanted to play nice.  If we want to reap the resourcegroup/cluster after an entire run then that's a :+1:  from me.